Izmenene (#16)
savannstm/sotws-translation: translation/maps.txt

\et[22]He's all\#sorts of\#trouble.<#>Он - сплошная\#катастрофа.
\et[011]If it\#ain't beer\#I don't\#want it.<#>Мне не нужно\#ничего, кроме\#пива.
\et[011]Eight drinks\#in, don't even\#feel buzzed.<#>Влил восемь\#банок, и даже\#не шатает.

> \et[011]Daddy always did\#say I had a high\#alcohol tolerance.<#>Отец всегда говорил,\#что у меня высокая\#толерантность к алкоголю.

Reviewer: Да никто не говорит так, бомжи тем более, кроме Сергея Симонова

\et[069]What did you think\#was in this bag?<#>Знаешь, что лежало\#в этом пакете?
\et[000]That's terrible.<#>Это ужасно.
\et[069]Darry,\#you know\#how it is.<#>Дэрри, ты\#знаешь,\#каково это.

> \et[069]You can\#help him\#to see,\#right?<#>Ты можешь\#помочь\#ему увидеть,\#верно?

Reviewer: Ну тут видеть используется в значении типа как "просветлеть", "начать ведать", я так подозреваю, так что оставь старое

Trade Rando Coin?<#>Обменять монету Рандо?
\et[0]Sounds like\#a fair trade.<#>Звучит, как честный\#обмен.
\et[0]Here.<#>Держи.

> \et[162]I knew\#you'd\#be cool.<#>Я знал, что\#ты верно\#поступишь.

Reviewer: Ну нет, это же диалог с этим как его, Зилфисом, он такой чилловый парень
Оставь старое

\et[02]Being a klutz will\#get you torn to\#shreds by rag\#maggots.<#>Промедлишь,\#и тряпичные\#опарыши\#разорвут тебя.
\et[07]The three of us\#can handle a few\#rag maggots,\#Georgy...<#>Втроём мы сможем\#справиться с несколькими\#тряпичными опарышами,\#Джорджи...
\et[02]Look after\#the place\#while I'm\#gone, Antuan.<#>Присмотри за\#домом, пока\#меня нет,\#Антуан.

> \et[04]Don't worry,\#putito. I will.<#>Не беспокойся,\#путито. Присмотрю.

Reviewer: Лос поиос эрманос

> \et[04]Don't worry,\#putito. I will.<#>Не беспокойся,\#пу'ито. Присмотрю.

Reviewer: ```suggestion
\et[04]Don't worry,\#putito. I will.<#>Не беспокойся,\#пуито. Присмотрю.
```

\et[02].\..\..\.Ugh...<#>.\..\..\.Уф...
\et[02]Alright.\.\.\#Fine.<#>Ладно.\.\.\#Хорошо.
\et[02]I'll go get\#things ready\#for departure.<#>Я приготовлю\#всё к отплытию.

> \et[02]Don't let your\#new friend\#keep you\#too long...<#>Не позволяйте\#вашему новому\#другу вас подолгу\#задерживать...

Reviewer: Вообще тут происходит диалог перед отплытием, что понятно из контекста, так что наверное "не позвольте вашему новому другу вас надолго задержать" или типа того подойдёт лучше

\et[000]To the mainland!<#>На большую землю!
<!-- Map020.rvdata2 --><#>Пещера в болотную часть мусорного острова
<!-- Order: 23 --><#>
\et[068]Where is Pedro!?\#I'm taking you all\#out with me!<#>Где Педро?!\#Я заберу всех\#вас с собой!
\et[068]With Rando gone,\.\#this island is\#Pedro's to rule!<#>Рандо умер, и теперь\#этим островом будет\#править Педро!
\et[068]There's nothing\#you trashheads\#can do to stop him!<#>Вам, бомжам,\#никак нас не\#остановить!
\et[068]Ha,\. wussies!<#>Ха,\. слабаки!

> \et[068]That'll teach\#you not to\#mess with Pedro!<#>Вбейте себе - \#не связываться\#с Педро!

Reviewer: Ну мне кажется тут всё таки без тире надо

> \et[068]That'll teach\#you not to\#mess with Pedro!<#>Вбейте себе - \#не связываться\#с Педро!

Reviewer: ```suggestion
\et[068]That'll teach\#you not to\#mess with Pedro!<#>Вбейте себе\#не связываться\#с Педро!
```

\et[068]Now get\#outta here!<#>Теперь\#убирайтесь!
\et[068]Go eat\#garbage\#or something!<#>Идите\#поешьте\#мусора!
\et[068]Just leave\#us alone!<#>Просто оставьте\#нас в покое!
He is dead.<#>Он мёртв.
\et[071]ESCAPING:<#>ПОБЕГ:
\et[071]Some fights allow the "escape"\#option for you and your friends.<#>Некоторые битвы позволяют вам, и\#вашим друзьям сбежать.
\et[071]If a fight is just too\#much, don't wait, run!<#>Если битва слишком тяжела,\#не ждите, бегите!

> \et[071]Running from fast enemies is hard.\#Don't give up on escaping if you fail!<#>От быстрых врагов убежать тяжелее.\#Не сдавайтесь, даже если не вышло!

Reviewer: ```suggestion
\et[071]Running from fast enemies is hard.\#Don't give up on escaping if you fail!<#>От быстрых врагов будет тяжелее сбежать.\#Не сдавайтесь, даже если не вышло!
```

\et[118]You'll never find\#another place good\#as this, buddy.<#>Ты никогда не найдёшь\#настолько же хорошее\#место, приятель.
\et[118]Just accept it.<#>Просто прими это.
\et[000]I wouldn't call\#this "acceptance,"\.\#rather\. "giving up."<#>Хотя я бы не назвал\#это "принятием",\.\#скорее "сдачей".

> \et[118]Call it what you\#want, that doesn't\#change the truth.<#>Называй, как хочешь,\#правды это не меняет.

Reviewer: Разве после "называй" нужна запятая?

\et[118]What kinda\#stunt are you\#tryna pull here?<#>Что за трюки\#ты здесь\#вытворяешь?

> \et[118]You say you\#don't want this,\.\#then hump bags\#like no tomorrow?<#>Ты говорил, что не хочешь,\.\#а теперь трахаешь\#пакеты так, будто завтра\#не наступит?

Reviewer: ```suggestion
\et[118]You say you\#don't want this,\.\#then hump bags\#like no tomorrow?<#>Ты говорил, что не хочешь,\.\#а теперь трахаешь\#пакеты так, будто завтра\#умрёшь?
```

\et[040]It'll be a pleasure\#travelling with you gentlemen.<#>Буду рад путешествовать с вами,\#джентльмены.
Geen Jean joined your party!<#>Гин Джин вступил в вашу группу!
\et[041]I hate everyone here.<#>Я ненавижу всех здесь.

> \et[041]Don't tell them\#I've said that.<#>Только им не говори этого.

Reviewer: ```suggestion
\et[041]Don't tell them\#I've said that.<#>Только им это не говори.
```

\et[059]Jumped the\#fishing platform\#like you could\#swim or somethin'...<#>Спрыгнул с рыбацкого\#плота, как-будто умеешь\#плавать...
\et[000]It was... an\#accident.<#>Это... вышло\#случайно.
\et[000]I got washed\#ashore.<#>Меня смыло\#на берег.

> \et[059]Hmph.<#>Пфф.

Reviewer: Ну он типа недоволен, такой ХММГГГ

I'm not buying that.<#>Я это не куплю.
You're carrying too much.<#>Ты слишком много несёшь.
Enjoy yourself.<#>Наслаждайся.

> Earth to dumbass, there's nothing there.<#>Земля вызывает долбоёба, ничего не осталось.

Reviewer: ```suggestion
Earth to dumbass, there's nothing there.<#>Приём, придурок, ничего не осталось.
```

\et[002]Well it ain't\#empty anymore\#and I'm not alone.<#>Ну, теперь она\#не пустая, а я не\#одинок.
\et[000]That's good, Joel.<#>Славно, Джоэл.
\et[002]It is and\#so is sleeping.<#>Славно было бы\#и поспать.

> \et[002]Jesus.<#>Иисусе.

Reviewer: Ну неее

\et[062]Before you\#ask,\. no, I don't\#have the watch.<#>Можешь не спрашивать -\.\#нет, часов у меня нет.
\et[062]Jewelry\#doesn't\#do it\#for me.<#>Ювелирка\#это не по\#мне.
\et[062]I passed\#the thing\#over to\#Howling Kebab.<#>Я передал их\#воющему\#кебабу.

> \et[062]He's over\#by the\#Black\#Swan.<#>Он у\#Чёрного\#лебедя.

Reviewer: Ну тут имеется в виду здание в виде чёрного лебедя, а не какой-то особый термин, так что маленькими

> \et[062]He's over\#by the\#Black\#Swan.<#>Он у\#Чёрного\#лебедя.

Reviewer: ```suggestion
\et[062]He's over\#by the\#Black\#Swan.<#>Он у\#чёрного\#лебедя.
```

<!-- Map137.rvdata2 --><#>У складки Бесконечности
<!-- Order: 176 --><#>
Some force beckons from beyond the wall.<#>Из-за стены исходит некая сила.
You hear a faint beating.<#>Вы чувствуете слабый пульс.
Silence.<#>Тишина.

> You hear a pounding heart.<#>Вы слышите биение сердце.

Reviewer: Опечатка

\et[059]Ain't nevah seen\#him so blue.<#>Не видал его\#таким синим.
\et[059]Ah, rest\#his soul.<#>Эх, упокоится\#его душа.
\et[059]You got\#da good,\#good smell.<#>Ты вкусно,\#вкусно\#пахнешь.

> \et[059]Anotha man\#enterin' da\#land o' honey.<#>Очередной мужчина\#входит в намазанные\#мёдом земли.

Reviewer: ```suggestion
\et[059]Anotha man\#enterin' da\#land o' honey.<#>Очередной человек\#входит в намазанные\#мёдом земли.
```

Broken bones.<#>Сломанный скелет.
Esoteric texts.<#>Эзотерические записи.
"9:31, 31/12/99"<#>"9:31, 31.12.99"
"The magnitude of this currently-\#ongoing event is unprecedented.<#>"Масштабы происходящего в настоящее\#время события беспрецедентны."

> "Nothing of this scale has ever\#been observed by our team.<#>"Ничто настолько же масштабное никогда не\#обнаруживалось нашей командой."

Reviewer: ```suggestion
"Nothing of this scale has ever\#been observed by our team.<#>"Ничто настолько же масштабное никогда не\#обнаружалось нашей командой."
```

\et[011]That ship sailed a\#loooooong time ago.<#>Этот лагерь затонул\#ооооочень давно.
\et[011]The other guys\#can sit on their\#asses as long\#as they like.<#>Пусть другие\#сидят на жопе\#ровно сколько\#хотят.
\et[011]I'm getting the\#hell out of here\#while I'm still\#alive to do it.<#>Я выбираюсь\#отсюда к чёрту,\#пока я ещё жив.

> \et[012]I'm on guard\#for any members\#of that red and\#green scourge.<#>Я охраняю лагерь\#от любых участников\#этой красно-зелёной\#беды.

Reviewer: ```suggestion
\et[012]I'm on guard\#for any members\#of that red and\#green scourge.<#>Я охраняю лагерь\#от участников\#этой красно-зелёной\#беды.
```

\et[000]Pft, some\#magician\#you are...<#>Пфф, ну\#и маг\#из тебя...
\et[000]...selling us a\#horribly overpriced\#first-aid kit.<#>...продающий\#дико дорогие\#аптечки.
\et[001]Please...\| don't\#tell anyone.<#>Пожалуйста...\| не\#говори никому.
\et[001]I have a real\#reputation\#around here.<#>У меня здесь\#хорошая\#репутация.

> \et[001]If they\#found out\#I was a\#fraud...<#>Если они\#узнают, что\#я мошенник...

Reviewer: Вообще, он не мошенник, он просто заявляет, что он шаман, лечащий всех магией, но на деле он просто хирург

\et[003]Let's just say\#that we tried\#doing this before.<#>Давай сойдёмся на\#том, что некоторые\#уже пытались.
\et[003]And something\#like that may or\#may not have happened.<#>И что-то подобное\#могло или не могло\#происходить раньше.
\et[000]Well,\.\. I\#did what\#I was told.<#>Ну,\.\. я сделал\#то, что было\#сказано.

> \et[000]They've\#all been\#given away.<#>Все монеты\#розданы.

Reviewer: Только не это слово

\et[003]Then\#goodbye,\. Alex.<#>Тогда прощай,\.\#Алекс.
\et[003]May you\#find your\#place someday.<#>Да найдёшь\#ты своё\#место.
\et[003]Godspeed,\.\#soldier.<#>С божьей помощью,\.\#солдат.

> Obliterated.<#>Стёрт.

Reviewer: Я всё думал, как написать "стереть" в этой форме, в итоге забил :trollface:
